guides/line-management/README.md

```diff
@@ -0,0 +1,35 @@
+# Line management at Made Tech
```

Comment:
This has been floated around before but maybe here is a good place to discuss -
The wording 'line management' leaves a lot to be desired, and has a connotation of management style attached to it.

I personally prefer the wording along the lines of `Career Mentor`. I think this still describes the responsibilities well, while removing some of the more negative associations such as performance management. It's not like a career mentor would shy away from/not be doing _management_ in some cases, but language is important and I feel a conscious effort to move in this direction would help build trust and open communication by presenting the career mentor as an ally and friend.

Reply:
@sarrionandia said something that really resonated with me a while ago on this - there's an 'honesty' in the term 'line manager', as it's someone who's responsible for things associated with that role traditionally, including the more difficult bits.

I hear you, though - I don't think we do this typically at Made Tech, and we miss that in the name.

Reply:
I don't disagree! I wonder if there's a dishonesty in wrapping those responsibilities in wording like mentor?

Could that have an adverse effect when actually doing the difficult bits? I had assumed not, that it may make the difficult bits easier to start on.

Reply:
I think 'Career Mentor' is a great name but in conflict with some of the aspects of line management that are not purely coaching/mentoring. Such as performance management. I think it could be misleading to call this role Career Mentor but it certainly is a big part of that role.

Although I'm also not a fan of the term Line Manager :) 
I have heard the word 'Sponsor' before but that might be unclear. Maybe we can review the role name as a group later

Reply:
I'd probably argue performance management is mentoring in a sense, but getting into semantics/obfuscating that probably isn't advisable